plugins/woocommerce/includes/class-wc-tracker.php

```diff
@@ -542,6 +543,39 @@ private static function get_orders_by_gateway() {
 		return $orders_by_gateway_currency;
 	}
 
+	/**
+	 * Get orders origin details.
+	 * 
+	 * @return array
+	 */
+	private static function get_orders_origins() {
+		global $wpdb;
+
+		$orders_origin = $wpdb->get_results(
+			"
+			SELECT
+				meta_value as origin, COUNT( DISTINCT ( orders.id ) ) as count
+			FROM
+				wp_posts orders
+			LEFT JOIN
+				wp_postmeta order_meta ON order_meta.post_id = orders.id
```

Comment:
We should avoid hard-coding the table prefix (within some installations a custom prefix may be defined). How about:

```suggestion
				$wpdb->posts orders
			LEFT JOIN
				$wpdb->postmeta order_meta ON order_meta.post_id = orders.id
```